charts/openshift-metering/templates/hive/hive-server-statefulset.yaml

```diff
@@ -142,8 +157,10 @@ spec:
           defaultMode: 0555
 {{- if .Values.hive.spec.config.useHadoopConfig }}
       - name: hadoop-config
+        emptyDir: {}
+      - name: hadoop-starting-config
         secret:
-          secretName: {{ .Values.hive.spec.config.hadoopConfigSecretName }}
+          secretName: {{ .Values.hive.spec.config.hadoopConfigSecretName }} 
```

Comment:
Extra space at end of line. You should really configure your editor to show/fix this automatically.